R/WIS_density_plot.R

```diff
@@ -0,0 +1,57 @@
+#' Title
+#'
+#' @param wis_data
+#'
+#' @return
+#' @export
+#'
+#' @examples
```

Comment:
Would it be possible to please add some documentation on the functions, so we can understand what it does and how to use it? It also will fix some Warnings in the R CMD check GitHub actions.